------------------------------------------------------------
revno: 251
committer: Scott Moser <smoser@ubuntu.com>
branch nick: trunk
timestamp: Fri 2010-08-13 11:15:58 -0400
message:
  uncloud-init: move marker file into /var/lib/cloud/sem
  
  All other marker files by cloud-init live in /var/lib/cloud/sem
  it makes sense for uncloud-init's file to live there too.
=== modified file 'tools/uncloud-init'
--- old/tools/uncloud-init	2010-08-12 05:56:12 +0000
+++ new/tools/uncloud-init	2010-08-13 15:15:58 +0000
@@ -10,11 +10,14 @@
 KEY="xupdate"
 UMOUNT=""
 RMDIR=""
-MARK=/root/uncloud-init-ran
+MARK=/var/lib/cloud/sem/uncloud-init.once
 ROOT_RW=""
 
 doexec() {
-	[ -z "$ROOT_RW" ] || date > "${MARK}";
+	if [ -n "$ROOT_RW" ]; then
+		mkdir -p "${MARK%/*}";
+		date > "${MARK}";
+	fi
 	cleanup;
 	log "invoking /sbin/init $*"
 	exec /sbin/init "$@";

